### DISCUSSION TITLE
Making your work accessible for all lawyers

### DISCUSSION THREAD

**Discussion by passamo9** [passamo9/LaTeX_EnglandWales_Templates#13] [Q&A]
I had a couple of questions on setting up the Documents as you have coded them. Could we have that discussion so that it becomes clear for other lawyers (that also don't have LaTeX experience) can use what you have created?

I also don't want to make this more complicated than it should be. Open to ideas to figure out how best everyone works.

**Comment by bond7o6** [comment]
We could also bundle all the information that gets put in this channel into documentation (if it gets made) at some point.

**Reply by passamo9** [reply]
That would be super useful! So that it's clear how it all works together

**Reply by Andonome** [reply]
That's what the wiki's for.

I can do a 'setup for Windows' and 'Setup for Mac' guide in the wiki (if someone's on Linux, they probably don't need the guide).

Maybe just:

1. Download thing (then make this a link to the latest .zip file which has the master branch files)
2. Install [ some very easy Latex thing - Texlive Studio maybe? ]
3. Screenshot of opening the file and pressing the 'compile' button
4. Instruction to edit the `claimant` and `defendent` text

I can do this in my one if someone here knows how to pull/ push Wikis, otherwise I'll need commit access here.

**Reply by passamo9** [reply]
That would be utterly amazing!

Thank you!

**Reply by passamo9** [reply]
Can you write the wiki here then I will upload it?

**Reply by bond7o6** [reply]
Yeah, the wiki is probably the best place for that info. I was thinking putting package tool usage (that are made in this project) into a documentation for ctan if it makes it to that point.

**Reply by passamo9** [reply]
That would be absolutely incredible!!

**Reply by bond7o6** [reply]
It looks like wiki changes by people without commit access isn't possible, or is at least undocumented since they changed it. If we want to use the wiki, it's probably best if you make an issue with the wiki page, then someone with commit access and check and move it to the wiki. The other way would be to have a `doc` folder with everything documentation-y stored in it, which would allow for PRs. Anyone have any preference?

**Reply by passamo9** [reply]
I've made the Issue for the Wiki page, calling it [Wiki Page - Guide for Lawyers](https://github.com/passamo9/LaTeX_EnglandWales_Templates/issues/20).

@yjyao has commit access, if I'm not mistaken.

>The other way would be to have a doc folder with everything documentation-y stored in it, which would allow for PRs. 

Please let me know if I should close the Wiki if anyone prefers it closed.

**Reply by bond7o6** [reply]
Ahh, I just made an issue as well.

**Reply by yjyao** [reply]
should we try it out with the [wiki repo](https://docs.github.com/en/github/building-a-strong-community/adding-or-editing-wiki-pages#adding-or-editing-wiki-pages-locally)? not sure if PRs come here as well. if PRs go to another page then we probably don't want it.

> @yjyao has commit access

yes i do now but i was thinking you should maybe remove me after #18 is merged?

**Reply by passamo9** [reply]
>should we try it out with the wiki repo?

I don't see why not

> was thinking you should maybe remove me after #18 is merged?

Yeah, if you think that's better

**Comment by Andonome** [comment]
So, in the wiki put this (but keep the formatting):

---

# Setup Instructions

1. Download the files [here](https://github.com/passamo9/LaTeX_EnglandWales_Templates/archive/main.zip)
2. Unzip the file
2. Download a LaTeX editor
   - Windows users might try [TexLive Studio](http://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe)
   - Mac Users might user [ idk - anyone know a good editor for mac? ]
3. Open the example document with your LaTeX editor.

![image](https://user-images.githubusercontent.com/52453904/107893545-2a120400-6f2c-11eb-9076-eaa4992a72f0.png)

Finally, press the 'Play' button to see your document:

![image](https://user-images.githubusercontent.com/52453904/107893954-44e57800-6f2e-11eb-80c5-7b0b3c8e5f54.png)

**Comment by Andonome** [comment]
Maybe the 'play' button could do with a red circle, or is that patronizing?

Also, I can't get this document working in Windows.  Maybe someone else can test?

And perhaps someone could add some resource for Macs - do they use a package manager?

**Reply by passamo9** [reply]
>Also, I can't get this document working in Windows. Maybe someone else can test?

I'm trying to figure this out too. I think it's going to click for me soon.

>And perhaps someone could add some resource for Macs - do they use a package manager?

I added MacTex, as this is what I am using right now.
